app/mailers/proposal_mailer.rb

```diff
@@ -20,4 +20,18 @@ def proposal_approved_notification(user, short_name, version, id)
 
     mail(to: "#{@user[:name]} <#{@user[:email]}>", subject: email_subject)
   end
+
+  def proposal_rejected_notification(user, proposal)
+    @user = user
```

Comment:
I like how you just pass the proposal object into this rather than all the individual fields, why kind of wondering if you should have done that for the other methods.